clang/test/Driver/sycl-device-optimizations.cpp

```diff
@@ -36,3 +36,12 @@
 // RUN:   | FileCheck -check-prefix=CHECK-DAE %s
 // CHECK-DAE: clang{{.*}} "-fenable-sycl-dae"
 // CHECK-DAE: sycl-post-link{{.*}} "-emit-param-info"
+
+// Check "-fgpu-inline-threshold" is passed to the front-end:
+// RUN:   %clang -### -fsycl -fgpu-inline-threshold=100000 %s 2>&1 \
+// RUN:   | FileCheck -check-prefix=CHECK-THRESH %s
+// CHECK-THRESH: "-mllvm" "-inline-threshold=100000"
```

Comment:
Should the documented default value of `225` be tested as well?

Reply:
Hmm, I guess this would have to be a different kind of test? As far as I can tell, if -fgpu-inline-threshold isn't specified, nothing is passed from the clang driver to LLVM.

Reply:
The default value of `225` is hard coded in `InlineCost.cpp` (llvm), it is not set by the clang driver. This is out of scope of this patch.

Reply:
Got it, thanks.